src/util/localStorage.native.ts

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import { IStorage } from '../types'
+
+let storage: IStorage = {
+  async clear(): Promise<void> {
+    await AsyncStorage.clear()
+  },
+
+  async getItem(key: string): Promise<string | null> {
+    return AsyncStorage.getItem(key)
+  },
+
+  async removeItem(key: string): Promise<void> {
+    storage = _.omit(storage, key)
+    await AsyncStorage.removeItem(key)
+  },
+
+  async setItem(key: string, value: string): Promise<void> {
+    storage[`${key}`] = value
+    await AsyncStorage.setItem(key, value)
```

Comment:
Can you explain the reason for having two different assignments, one to a plain object and one to `AsyncStorage`?

Reply:
There were some places where you were accessing the localStorage value using the storage object. Like: ` storage['Settings/' + value]`. I think I will just refactor it to access using the `getItem` method so we avoid double assignment.

Reply:
Yes. `localStorage` is special in that assignment is equivalent to `setItem`. We should use `getItem` and `setItem` instead.

Reply:
Fixed.